frontend/src/dto/ApplyClientNumberDto.ts

```diff
@@ -9,20 +9,17 @@ export interface FormDataDto {
     incorporationNumber: string;
     businessName: null | string;
     goodStandingInd: string;
-    firstName: string;
-    lastName: string;
-    birthdate: string;
-    doingBusinessAsName: string;
-  };
+    fullMatchInd: string;
```

Comment:
What is the purpose of this parameter? As of now, when requesting the details, an error will be thrown when a full match is found

Reply:
As per our conversation, I removed it as we don't longer needed it